scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1309,3 +1312,89 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
   </tbody>
 </table>
 {% endmacro %}
+
+{% macro somatic_variant_table(variants) %}
+  <div class="card" style="border-width: 5px; display: block;">
+    <div class="card-header">Summary table</div>
+    <div class="card-body">
+      <div class="row d-flex align-items-center">
+       <table id="panel-table" class="table table-sm table-bordered" style="background-color: transparent">
+        <thead>
+          <tr class="table-secondary">
+            <th>Gene Symbol</th>
+            <th>Transcript</th>
+            <th>HGVS c.</th>
+            <th>HGVS p.</th>
+            <th>Tumor VAF%</th>
```

Comment:
Isn't VAF specific for SNVs?

Look at the example, the second variant is a SV. 

<img width="866" alt="image" src="https://user-images.githubusercontent.com/28093618/228197471-1db995c1-a1f4-4697-a9e8-a73dc72babda.png">

Reply:
It's a mixed bag for SVs: VAF could be rather better calculated than for SNVs for copy number variants but would be very difficult to compute for some balanced or semi-balanced events. So, good point. I'll add some of the alt line exceptions to the standard lines, like print sub-category if it's not "SNV".

Reply:
Like so:
![Screenshot 2023-03-28 at 12 10 18](https://user-images.githubusercontent.com/758570/228204146-78154a35-84d9-4232-9ccb-7706b4556939.png)

Reply:
Rounding fix:
![Screenshot 2023-03-30 at 10 58 44](https://user-images.githubusercontent.com/758570/228785801-47e04af6-c7c9-49e6-9a8a-539e784402a3.png)